react/src/pages/StartPage.tsx

```diff
@@ -20,10 +22,32 @@ const StartPage: React.FC = () => {
   const baiClient = useSuspendedBackendaiClient();
   const blockList = baiClient?._config?.blockList ?? [];
   const inactiveList = baiClient?._config?.inactiveList ?? [];
+  const enableModelFolders = baiClient?._config?.enableModelFolders ?? false;
 
   const webuiNavigate = useWebUINavigate();
   const [isOpenCreateModal, setIsOpenCreateModal] = useState<boolean>(false);
 
+  const { upsertNotification } = useSetBAINotification();
+  const { count } = useVFolderInvitationsValue();
+
+  useEffect(() => {
+    if (count > 0) {
```

Comment:
Fail Fast pattern would be better 




```suggestion
    if (count <= 0) return;
```